<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1171</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="NavBar" name="navBar" native="true">
      <property name="minimumSize">
       <size>
        <width>61</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>61</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1108</width>
         <height>665</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QStackedWidget" name="stackedWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="page">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item alignment="Qt::AlignmentFlag::AlignVCenter">
             <widget class="QWidget" name="widgetLogo" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QLabel" name="label">
                 <property name="font">
                  <font>
                   <pointsize>18</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string># Fuhrpark Explorer</string>
                 </property>
                 <property name="textFormat">
                  <enum>Qt::TextFormat::MarkdownText</enum>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignmentFlag::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>#### acht Tage die Woche Fullsupport!</string>
                 </property>
                 <property name="textFormat">
                  <enum>Qt::TextFormat::MarkdownText</enum>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignmentFlag::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="logo">
                <property name="text">
                 <string/>
                </property>
                <property name="pixmap">
                 <pixmap resource="../assets/ressources.qrc">:/cars/audi v13 tuning sportwage.png</pixmap>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="RentFormular" name="rentFormular"/>
          <widget class="QWidget" name="wg_Fahrzeug">
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QSplitter" name="splitterCarDisplay">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="handleWidth">
               <number>1</number>
              </property>
              <widget class="NavigationTree" name="navigationTree" native="true"/>
              <widget class="CarDisplayWidget" name="carDisplay" native="true">
               <property name="styleSheet">
                <string notr="true">border-left: 1px solid rgba(0,0,0,50);</string>
               </property>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="CarStatus" name="carStatus"/>
          <widget class="RentedCarsByCustomers" name="rentedCardsByCustomers"/>
          <widget class="History" name="history"/>
          <widget class="Statistic" name="statistic"/>
          <widget class="Customers" name="customers"/>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1171</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actQuit"/>
   </widget>
   <widget class="QMenu" name="menuAnsicht">
    <property name="title">
     <string>Ansicht</string>
    </property>
    <addaction name="actNavigationOnOff"/>
   </widget>
   <widget class="QMenu" name="menuHinzuf_gen">
    <property name="title">
     <string>Hinzuf√ºgen</string>
    </property>
    <addaction name="actAddNewCar"/>
    <addaction name="separator"/>
    <addaction name="actCustomer"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuAnsicht"/>
   <addaction name="menuHinzuf_gen"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actQuit">
   <property name="text">
    <string>Beenden</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actNavigationOnOff">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Navigation an/aus</string>
   </property>
  </action>
  <action name="actAddNewCar">
   <property name="text">
    <string>Fahrzeug</string>
   </property>
  </action>
  <action name="actCustomer">
   <property name="text">
    <string>Kunde</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NavigationTree</class>
   <extends>QWidget</extends>
   <header>app.widgets.NavigationTree</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CarDisplayWidget</class>
   <extends>QWidget</extends>
   <header>app.widgets.CarDisplayWidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RentFormular</class>
   <extends>QWidget</extends>
   <header>app.widgets.RentFormular</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CarStatus</class>
   <extends>QWidget</extends>
   <header>app.widgets.CarStatus</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RentedCarsByCustomers</class>
   <extends>QWidget</extends>
   <header>app.widgets.RentedCarsByCustomers</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Statistic</class>
   <extends>QWidget</extends>
   <header>app.widgets.Statistic</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>History</class>
   <extends>QWidget</extends>
   <header>app.widgets.History</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Customers</class>
   <extends>QWidget</extends>
   <header>app.widgets.Customers</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NavBar</class>
   <extends>QWidget</extends>
   <header>app.widgets.bars.NavBar</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../assets/ressources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>487</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
